### ISSUE TITLE
Add German correctional facilities ("Vollzugsanstalten")

### ISSUE THREAD

**Issue by m-rey** [datenanfragen/data-imports#2]
While verifying data for datenanfragen/data#1634, I've found an official looking [list of all German correctional facilities](https://www.justiz.nrw/Gerichte_Behoerden/anschriften/justizanschriften/bundesweit/justizvollzugsbehoerden.pdf) (="Vollzugsanstalten") on https://www.justiz.nrw/Gerichte_Behoerden/anschriften/justizanschriften/index.php.

This could possibly be used to (semi) automatically, or at least more easily, add missing German correctional facilities to the dataset.

**Comment by m-rey** [comment]
I agree that the PDF file by itself doesn't contain enough data. We would need to find additional/better data sources before it can be added to the dataset.
I noticed that the list seems to be quite complete. We could use it as a basis and add missing data protection-specific contact points. Maybe we could even find a better list with the information we need on it, using data from this PDF file.

**Comment by baltpeter** [comment]
One idea on how we could get a better list would be through FOI (IFG) requests to the state data protection authorities. @zner0L tried to do that a while ago for all companies but abandoned the request due to the high cost he was asked to pay. If we scope the request to only correctional facilities (and can even provide a list of the ones in that particular state), I could imagine this falling under the scope of a simple request and ending up free.
